ui/src/components/overview/Overview.js

```diff
@@ -67,6 +81,8 @@ class Overview extends Component {
             mobileClients={mobileClients}
             mobileServiceInstances={mobileServiceInstances}
             mobileClientBuilds={mobileClientBuilds}></MobileClientOverviewList>
+        <MobileClientOverviewList mobileClients={this.state.mobileClients}></MobileClientOverviewList>
```

Comment:
this line should not be here. Looks like a merge error.

Reply:
yes